builder/scaleway/config.go

```diff
@@ -160,9 +225,9 @@ func (c *Config) Prepare(raws ...interface{}) ([]string, error) {
 	}
 
 	if errs != nil && len(errs.Errors) > 0 {
-		return nil, errs
+		return warnings, errs
```

Comment:
why do you return warnings here?

Reply:
Returning warnings even on error seems legit no? 😄